hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3677,7 +3682,16 @@ public boolean getEnableFileSystemPaths() {
   }
 
   public ReplicationConfig getDefaultReplicationConfig() {
-    return ReplicationConfig.getDefault(configuration);
+    final String replication = configuration.get(
+        OZONE_SERVER_DEFAULT_REPLICATION_KEY,
+        OZONE_SERVER_DEFAULT_REPLICATION_DEFAULT
+    );
+    final String type = configuration.get(
+        OZONE_SERVER_DEFAULT_REPLICATION_TYPE_KEY,
+        OZONE_SERVER_DEFAULT_REPLICATION_TYPE_DEFAULT
+    );
```

Comment:
Shouldn't we pass the same configuration below 3rd arg?

Reply:
configuration.get only takes 2 arguments.

Reply:
We don't need, since the `replication` and `type` can't be `null`.
And it's also weird to fallback to client side config here.

Reply:
Oops, we still need to use the validator in the config.